.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+# Overview
+A high level description of the contribution, including:
+Who the change affects or is for (stakeholders)?
+What is the change? 
+Why is the change needed?
+How does this change affect the user experience (if at all)?
+
+# Design of Change
+How was this change implemented?
+
+# Related Issues/Pull Requests
+[ ] [Issue #1234](https://github.com/hashicorp/vault/issues/1234)
+[ ] [PR #1234](https://github.com/hashicorp/vault/pr/1234)
```

Comment:
I'm not sure why, but this is showing up on the same line as the previous checkbox in the rich view

Reply:
I think this needs to be in an unordered list:
```
- [ ] ...
```

Reply:
I think it's just the preview that's off

Reply:
It looks off when I view the file directly: https://github.com/hashicorp/vault-plugin-auth-alicloud/blob/ccfabfd502c5d7edb663511e90f176ebcb677675/.github/PULL_REQUEST_TEMPLATE.md


<img width="450" alt="Screen Shot 2020-03-30 at 4 41 21 PM" src="https://user-images.githubusercontent.com/72365/77968746-4e79d900-72a5-11ea-997d-cde9e34e2498.png">

Reply:
But it does look OK when filling out the PR AFAICT.